Python/OTP_verifier.py

```diff
@@ -0,0 +1,15 @@
+import random
+from twilio.rest import Client
+
+otp=random.randint(1000,9999)
+print(otp)
+account_sid='Find from your twilio account'
+auth_token='Finfd from your twilio account'
+client=Client(account_sid,auth_token)
+
+message=client.message.create(
```

Comment:
The Code seems to be unformatted. Format your Code properly and indent it for better understanding.